week3/README.md

```diff
@@ -0,0 +1,61 @@
+#### Part 1 - Short-answer questions.
+a) 0…5 and 0..<5 are two types of **ranges**. How are they different?</br>
+
+**`0...5` is a *closed range*; it includes both the lower and upper boundaries of the range (in this case: 0, 1, 2, 3, 4, 5).**
+
+**`0..<5` is a *half-open range*; it includes only the lower boundary of the range and not the upper boundary (in this example: 0, 1, 2, 3, 4).**
+
+b) Describe type inference in Swift and give an example.</br>
+**Type inference refers to the Swift compiler's ability to detect the type of a variable or constant based on its value. For example, I could declare a variable named `petAge` and set it to a value of `13`, like so: `var petAge = 13`. Swift would be able to infer that the type of my variable was an `Int`, because I set it to a numeric value with no decimal point.**
+
+c) List three advantages of Playgrounds.
+- **You can set Playground code to auto-run and evaluate and display expression results in real-time (every time you make a change), so you can quickly run experiments with your code (tweaking functions, trying out different input parameter values, etc.)**
```

Comment:
Rapid prototyping.
Quick Feedback.
No need to make full Xcode project.

Reply:
I wish I were able to come up with such concise answers! You’re really good at it.